registry/recipes/osx-setup-guide.md

```diff
@@ -52,14 +52,14 @@ If you want to understand, you should read [How to Write Go Code](https://golang
 Copy the registry configuration file in place:
 
     mkdir /Users/Shared/Registry
-    cp docs/osx/config.yml /Users/Shared/Registry/config.yml
+    cp docs/recipes/osx/config.yml /Users/Shared/Registry/config.yml
```

Comment:
This should be upstreamed to https://github.com/distribution/distribution

Reply:
Looks like it's fixed upstream - https://github.com/distribution/distribution/blob/main/docs/recipes/osx-setup-guide.md#run-the-registry-under-launchd